BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
No need for .bazel btw, it's more traditional to not have this

Reply:
Also in the test/ folder

Reply:
I had to end up adding that to get the linter in VSCode to pick it up? Maybe that was just a me problem tho.

Reply:
You're probably using the wrong extension. Use the official VScode extension, not the one by Devon Crew.

Reply:
I have the official one. Now it's not having an issue picking up a `BUILD` or `WORKSPACE` file though so might have just been something weird because I hadn't restarted.
I updated the names tho!